packages/picasso/src/PageSidebar/PageSidebar.tsx

```diff
@@ -114,6 +83,10 @@ export const PageSidebar = forwardRef<HTMLDivElement, Props>(function Sidebar(
     }
   }, [setHasSidebar])
 
+  useEffect(() => {
+    setShowHamburger?.(true)
```

Comment:
nit: I'm not a fan of conditional calling functions like this due to decrease readability. I would prefer 

```suggestion
    if (setShowHamburger) {
        setShowHamburger(true)
    }
```

I remember discussing this with Daniel and he also said that it is not that common to use this conditional function call approach, so I have an architect opinion at least 🙂 still up to you 🙇‍♂️

Reply:
Done 👍